planning/Discovery/work-items.md

```diff
@@ -0,0 +1,51 @@
+# Web of Things (WoT) Discovery
+## Work Items
+Prioritized list of work items related to Discovery.
+- Detailed descriptions of work items will be in a single list (in a separate document) to accomodate
+  work items that impact more than one deliverable.
+- Each work item has a link to its detailed description, and a link to the requirement(s) it satisfies or helps satisfy.
+- Note that requirements will also link in turn to the use cases that motivate it.
+- Priority is given by the ordering in the list, from most to least important.
+     - No ties allowed; this is called force-ranking and is intentional.
+ 
+### To do:
+- Assign "categories" to work items to cluster them into a smaller number of "feature sets".
+    - Right now have categories for "Direct" (internal Discovery feature),
+      "Dependency" (involved in more than one deliverable), and "Process" (not really a spec feature,
+      but a procedural matter).
+- Following not yet actually prioritized, we may need to split into smaller work items first
+- Update base URL to work-item list after it gets moved
+- Need to add requirements and links from each work item, below only a couple of examples are given
+     - Link to requirements in wot-architecture
+     - We could also put these in a document, or in MD files under each deliverable
+     - However, a single list of requirements would allow us to address requirements that span more than one deliverable
+
+### Prioritized List
+1. Direct - [Discovery Improvements](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#discovery-improvements-workitem)
+    * Needs to be broken into smaller work-items to better support prioritization.
+2. Direct - [Discovery of Thing Models](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#discovery-thing-models-workitem) 
```

Comment:
A ThingModel cannot be discovered since it cannot be advertised as-is. I'd rather have it to the lowest priority since it would require overhauling a bit more TM before discussing how to advertise and how to put in the directory.

Reply:
I agree that this priority level for TMs discovery needs to be supported by some explanation. As far as I know, we don't have strong use cases besides the nice-to-have ability to link TMs using relative links in TDDs.

Reply:
Agree this can be lower in priority.  Right now the list has not yet been prioritized, it's just in the order they appeared in the work items.   But we would need to understand the use case for putting TMs in directories.  I can think of a couple use cases but they could be satisfied in other ways.